src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -144,7 +144,13 @@ def default_api_version(self):
             'role_definitions': '2018-01-01-preview',
             'provider_operations_metadata': '2018-01-01-preview'
         }),
-        ResourceType.MGMT_CONTAINERREGISTRY: '2019-06-01-preview',
+        ResourceType.MGMT_CONTAINERREGISTRY: SDKProfile('2019-12-01-preview', {
```

Comment:
well. I cannot find such api-version in https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/containerregistry/azure-mgmt-containerregistry/azure/mgmt/containerregistry.

Reply:
The SDK publishing is pending.
I have marked this PR as "Do not merge" to prevent any incidental merge